test/ImportCommand.js

```diff
@@ -57,6 +57,35 @@ describe("ImportCommand", () => {
       expect(await pathExists(packageJson)).toBe(true);
     });
 
+    it("imports a repo with conflicted merge commits when run with --flatten", async () => {
+      const cwdExternalDir = { cwd: externalDir };
+      const branchName = "conflict_branch";
+      const conflictedFileName = "conflicted-file.txt";
+      const conflictedFile = path.join(externalDir, conflictedFileName);
+      await fs.writeFile(conflictedFile, "initial content");
```

Comment:
Please put newlines before the `fs.writeFile()` bits so the logical steps are a bit easier to follow, thanks.